pages/my-dashboard.js

```diff
@@ -3,15 +3,15 @@ import { Heading } from '@dts-stn/service-canada-design-system'
 import en from '../locales/en'
 import fr from '../locales/fr'
 import Card from '../components/Card'
-import { getHomeContent } from '../graphql/mappers/home'
+import { getMyDashboardContent } from '../graphql/mappers/my-dashboard'
 import logger from '../lib/logger'
 
 export default function Home(props) {
```

Comment:
I'd like to also name the function MyDashboard. Just for the sake of consistency. :)

Reply:
Great catch!